index.html

```diff
@@ -1,57 +1,230 @@
-        <a  class="hamburguer"><img src="img/Enabled.png" alt="sub-menu" class="H-iamge"></a> 
-      </nav>
-      <div class="header-2">
+    <nav class="app-bar">
+      <a class="My-logo">My Logo</a>
+      <a class="hamburguer"><img src="img/Enabled.png" alt="sub-menu" class="H-iamge"></a>
+    </nav>
+    <div class="header-2">
       <section class="TXTPrimay">
         <div class="modal__header flex">
           <h1 class="Carlos">I’m Carlos.
-           Glad to see you!</h1>
+            Glad to see you!</h1>
           <p id="description">
-          I’m a software developer! I can help you build a product , feature or website Look through some of my work and experience! If you like what you see and have a project you need coded, don’t hesitate to contact me. 
+            I’m a software developer! I can help you build a product , feature or website Look through some of my work
+            and experience! If you like what you see and have a project you need coded, don’t hesitate to contact me.
           </p>
           <h2 id="connect" class="connect-title">LET’S CONNECT</h2>
-         
+
         </div>
-        </section>
-        <section id="icons-secction">
-          <ul class="social-media-icons">
-      <li class="icons">
-        <a href="#" target="_blank"><img src="img/twitter.png" class="img-icons"></a>
-      </li>
-       <li class="icons">
-        <a href="#" target="_blank"><img src="img/Linkedin icon.png" class="img-icons"></a>              
-      </li>
-      <li class="icons">
-        <a href="#"><img src="img/M.png" class="img-icons"></a>             
-      </li>
-      <li class="icons">
-        <a href="#" target="_blank"><img src="img/GitHub.png" class="img-icons"></a>              
-      </li>
-      <li class="icons">
-        <a href="#"><img src="img/Paz.png" class="img-icons"></a>
-      </li>
-         
-      </ul>
       </section>
+      <section id="icons-secction">
+        <ul class="social-media-icons">
+          <li class="icons">
+            <a href="#" target="_blank"><img src="img/twitter.png" class="img-icons"></a>
+          </li>
+          <li class="icons">
+            <a href="#" target="_blank"><img src="img/Linkedin icon.png" class="img-icons"></a>
+          </li>
+          <li class="icons">
+            <a href="#"><img src="img/M.png" class="img-icons"></a>
+          </li>
+          <li class="icons">
+            <a href="#" target="_blank"><img src="img/GitHub.png" class="img-icons"></a>
+          </li>
+          <li class="icons">
+            <a href="#"><img src="img/Paz.png" class="img-icons"></a>
+          </li>
+
+        </ul>
+      </section>
+
+
+    </div>
+  </div>
+  <div class="Works">
+    <section class="screen-shot ">
+      <img src="img/Snapshoot Portfolio.png" alt="project-image" class="screenshot-img">
+    </section>
+    <section class="text-block">
+      <header class="header-3">
+        <h2 class="project-tittle">Tonic</h2>
```

Comment:
[OPTIONAL] I think it is better to wrap all your works in one ``section`` tag as this allows you to organize and group your HTML better making it more accessible 😉